internal/packageutil/package.go

```diff
@@ -25,7 +25,8 @@ func IsCocoaApplicationPackage(p string) bool {
 	return strings.HasPrefix(p, "/private/var/containers") ||
 		strings.HasPrefix(p, "/var/containers") ||
 		strings.Contains(p, "/Developer/Xcode/DerivedData") ||
-		strings.Contains(p, "/data/Containers/Bundle/Application")
+		strings.Contains(p, "/data/Containers/Bundle/Application") ||
```

Comment:
should this be prefix instead of contain?
```suggestion
		strings.HasPrefix(p, "/data/Containers/Bundle/Application") ||
```

Reply:
I can't suggest the line above since it's not in the diff but same applies there

Reply:
I won't touch this logic since we're not 100% certain they are actually prefix (the line above is not for example).